lib/commands/init/helper.js

```diff
@@ -3,6 +3,7 @@ const path = require('path');
 const R = require('ramda');
 
 const SkillInfrastructureController = require('@src/controllers/skill-infrastructure-controller');
+const CliWarn = require('@src/exceptions/cli-warn');
```

Comment:
A little strange to have CliWarn as exception class. Exception classes should indicate type of exception. Warning is about how a message is displayed. At the same time, CliWarn was added 9 days ago, so it is not directly related to this PR. We may be going with a strange path with error handling again.

Reply:
So `warn` here is not exception for sure, it's a special error that should continue the execution. That's why CliWarn inherits CliError as they are used to denote the error type. I think with the existence of this type of error we can only manage the flow by the type of it.